.github/workflows/devbuild.yml

```diff
@@ -8,6 +8,69 @@ on:
 
 jobs:
 
+  standalone_buffer:
+
+    if: ${{ github.event_name != '' || (github.event_name == '' && github.repository_owner == 'solvcon') }}
+
+    name: standalone_buffer_${{ matrix.os }}
```

Comment:
Make sure it is always possible to build the buffer as a header-only sub-system standalone.